app/models/clinical_resource_hub.rb

```diff
@@ -0,0 +1,10 @@
+class ClinicalResourceHub < ApplicationRecord
+  # extend FriendlyId
```

Comment:
nit: can we remove the commented out code

Reply:
yep..